recipes/skbase/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "skbase" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}-suite
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/skbase-{{ version }}.tar.gz
+  sha256: 0f1c113fe094725f22619b30c6c8093ec77027d937adf1c2246de9344f10228e
+
+build:
+  number: 0
+  # noarch: python
+
+requirements:
+  build:
+    - python
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+outputs:
+  - name: {{ name|lower }}
+    script: build_script.sh  # [unix]
+    script: build_script.bat  # [win]
+    requirements:
+      build:
+        - python
+      host:
+        - python >=3.7,<3.12
+        - pip
+      run:
+        - python >=3.7,<3.12
+        - scikit-learn <1.3.0,>=0.24.0
```

Comment:
```suggestion
        - scikit-learn >=0.24.0,<1.3.0
```

Reply:
Thanks for pointing this out. I've resolved it in changes I have pushed.